adsws/feedback/views.py

```diff
@@ -86,50 +99,128 @@ def prettify_post(post_data):
+        # (currently, this only happens in the missing_references.txt template)
+        body = body.replace('[tab]','\t')
+        
+        return body        
+
+    @staticmethod
+    def process_feedbackform_submission(post_data, body):
+        """
+        Takes the data from the feedback forms and fills out the appropriate template
+        :param post_data: the post data to fill out email template, dictionary expected
+        :param body: email body
+        :return: success message, string
+        """
+        # List to hold attachments to be sent along
+        attachments=[]
+        if post_data.get('_subject') == 'New Record':
+            attachments.append(('new_record.json', post_data['new']))
+        if post_data.get('_subject') == 'Updated Record':
+            attachments.append(('updated_record.json', post_data['new']))
+            attachments.append(('original_record.json', post_data['original']))
+
+        feedback_email = 'no email sent'
+        if post_data.has_key('email') and post_data.has_key('name'):
+            try:
+                res = send_feedback_email(post_data['name'], post_data['name'], post_data['_subject'], body, attachments=attachments)
+                feedback_email = 'success'
+            except Exception as e:
+                current_app.logger.info('Sending feedback mail failed: %s' % str(e))
+                feedback_email = 'failed'
+
+        return feedback_email
+    
     def post(self):
         """
         HTTP POST request
         :return: status code from the slack end point
         """
 
         post_data = get_post_data(request)
+
         current_app.logger.info('Received feedback: {0}'.format(post_data))
 
+
         if not post_data.get('g-recaptcha-response', False) or \
                 not verify_recaptcha(request):
             current_app.logger.info('The captcha was not verified!')
             return err(ERROR_UNVERIFIED_CAPTCHA)
         else:
             current_app.logger.info('Skipped captcha!')
+        origin = post_data.get('origin', 'NA')
+        if origin == current_app.config['FEEDBACK_FORMS_ORIGIN']:
+            current_app.logger.info('Received data from feedback form "{0}" from {1} ({2})'.format(post_data.get('_subject'), post_data.get('name'), post_data.get('email')))
+            try:
+                email_body = self.create_email_body(post_data)
+            except Exception as error:
+                current_app.logger.error('Fatal error creating email body: {0}'.format(error))
+                return err(ERROR_EMAILBODY_PROBLEM)
+            try:
+                email_sent = self.process_feedbackform_submission(post_data, email_body)
+            except Exception as error:
+                current_app.logger.error('Fatal error while processing feedback form data: {0}'.format(error))
+                return err(ERROR_FEEDBACKFORM_PROBLEM)
+            if not email_sent:
```

Comment:
`process_feedbackform_submission` returns a string saying `success`, `failed`, ... but here, this if statement seems to expect a boolean. I would change the function to return a boolean.